### DISCUSSION TITLE
Customizing save notification in Relation Managers

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#3063] [Help] [answered]
It is not precised in the documentation where put the instruction below. In which file and method ?
CreateAction::make()
    ->successNotificationMessage('User registered')

**Comment by danharrin** [comment, marked as answer]
The relation manager has a `table()` method. It will contain a `headerActions()` method which contains the `CreateAction`.

**Reply by eaziweb95** [reply]
I dont understand could you give me an example please ?

**Reply by danharrin** [reply]
<img width="1210" alt="Screenshot 2022-07-08 at 19 44 18" src="https://user-images.githubusercontent.com/41773797/178052067-122037e7-8d1e-4f78-93d2-2c107a411b6b.png">

**Reply by eaziweb95** [reply]
Well, thanks a lot !

**Reply by eaziweb95** [reply]
I have the following message ...
Class "Filament\Tables\Actions\CreateAction" not found

**Reply by danharrin** [reply]
You're not on the latest version then

**Reply by eaziweb95** [reply]
the successNotificationMessage is deprecated ... in the filament 2.16 ... 
what is the good method ?

**Reply by danharrin** [reply]
`successNotificationTitle()`, its in the docs

Both methods still work though
